ambari/src/main/java/io/brooklyn/ambari/AmbariConfig.java

```diff
@@ -0,0 +1,36 @@
+package io.brooklyn.ambari;
```

Comment:
I don't like having this object but will plan to remove when we implement retrofit rather than cleanup now